### PR TITLE
Add tea party event

### PR DESCRIPTION
**Pull request by meltedesco**

### CODE REVIEW

File: content/W2015/W15-event-teaparty.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+There will be tea, pastries, and other snacks! Come out and socialize with younger and older WiCS members.
+
+Hope to see you there!
+
+## Event Details ##
+
++ **Who:** Big and Little CSters
++ **What:** Tea Party Social
++ **Where:** CS lounge (DC 3336)
```
Comment (ehashman) [inline content/W2015/W15-event-teaparty.md]: CS Lounge (capital L)

File: content/W2015/W15-event-teaparty.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+Slug: tea-party 
+Author: Melissa Tedesco 
+Summary: We'll be meeting up in the CS lounge for a tea party social!
+
+Big and Little CSters, we would like to invite you to our CSters Tea Party!
+
+There will be tea, pastries, and other snacks! Come out and socialize with younger and older WiCS members.
+
+Hope to see you there!
+
```
Comment (ehashman) [inline content/W2015/W15-event-teaparty.md:16]: Did we want to add an "About Big CSters" bit here?

File: content/W2015/W15-event-teaparty.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+Title: Big CSters Tea Party Social
+Date: 2015-01-25 16:00
+Category: Events
+Tags: social
```
Comment (ehashman) [inline content/W2015/W15-event-teaparty.md]: I think this should go in a separate commit, but I recommend adding a "big csters" label to this article and the board games one!

**Comment by ehashman** [comment]
Nice work, Melissa!

File: content/W2015/W15-event-teaparty.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+Slug: tea-party 
+Author: Melissa Tedesco 
+Summary: We'll be meeting up in the CS lounge for a tea party social!
+
+Big and Little CSters, we would like to invite you to our CSters Tea Party!
+
+There will be tea, pastries, and other snacks! Come out and socialize with younger and older WiCS members.
```
Comment (ehashman) [inline content/W2015/W15-event-teaparty.md]: Oh, uh, also, something about Bring Your Own Mug.

Not sure about "younger" and "older", lol :)

File: content/W2015/W15-event-boardgames.md [hunk 5]
```diff
@@ -19,3 +19,6 @@ Hope to see you there!
 + **What:** Board Games Meet and Greet
 + **Where:** Math CnD (MC 3002)
 + **When:** Thu. Jan. 8, 7:00&ndash;11:00PM
+
+# About Big CSters #
+Big CSters is our mentorship program for female Computer Science students. We host social activities throughout the term for CSters to meet new friends in all stages of university life.
```
Comment (ehashman) [inline content/W2015/W15-event-boardgames.md:24]: Hm, I would change this description. I don't know if it encompasses everything we want in our CSters program (mentoring being the big one); also, @annalorimer and I had discussed using more inclusive language, since we're not just limiting to cis-women in CS. Maybe you should leave this commit for later.

Comment (meltedesco) [inline content/W2015/W15-event-boardgames.md:24]: I agree that the description needs some re-wording. I just adapted the wording from the [Big CSters page](https://cs.uwaterloo.ca/wics/mentoring) on the CS website.

I think the discussion about what sort of language to use needs to be in a meeting and added to our Code of Conduct. Then, we'll take cues from there as to the language to use in ads. Once that is ratified, I'll come back to this and change the language. :)

**Comment by ehashman** [comment]
I'm going to close this PR; for the other change re: Big CSters tags and a blurb, please open a new one :)
